### PR TITLE
Add codeowners and workflow

### PR DESCRIPTION
**Pull request by nikolauska**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @eliasvaltteri @henrinie @michalksiezuk-nc @jererobles @vastamaki @nikolauska @biphobe @ViktorHegyvari @aashirjaved @Nissinen @PerttuSavolainen @lintula @iriekun
```
Comment (jererobles) [inline .github/CODEOWNERS:1]: Maybe add @terotil and the now-confirmed @katjajohanna :)

**Review by jererobles — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (nikolauska) [reply .github/CODEOWNERS]: Good point, added

Comment (PerttuSavolainen) [reply .github/CODEOWNERS]: NOTE we have a limit of 15 reviewers, did we go past that already?

Comment (PerttuSavolainen) [reply .github/CODEOWNERS]: it seems to be exactly 15 now. 😬

Comment (nikolauska) [reply .github/CODEOWNERS]: I hope we will in the future start to split it up a bit so different people are responsible for different things. We could probably discuss about that at some point if more functionality is being added

**Pull request merged**